ckan/setup/ckan_setup.sh

```diff
@@ -29,11 +29,11 @@ do
 done
 
 echo "Enabling debug mode"
```

Comment:
See lines 19-26 in this file, I think we'll have to do something a little different...